### PR TITLE
Update WinWizard to version 5.0.3

### PR DESCRIPTION
**Pull request by lukaszgo1**
This pull request   updates link to the WinWizard to the version 5.03. which is compatible with NVDA  2021.1.
https://github.com/lukaszgo1/winWizard/tree/V5.0.3

No code changes aside from updating last tested NVDA version were necessary however I've updated the readme to note that the new release is compatible with 2021.1 and I would be grateful if it could be updated on the website. Alternatively I'm  happy to do it myself if I'm told how to do so.

### CODE REVIEW

**Review by nvdaes — APPROVED** [review]

**Comment by nvdaes** [comment]
Hello @lukaszgo1 :  
Once this PR is merged, if you want to update the add-on documentation on the website yourself, please join the translations mailing list and request an invitation to the repo. Then you can checkout it, see the website/addons folder and update the .mdwn file.

Please see more info at

https://github.com/nvaccess/nvda/wiki/TranslatingUsingAutomaticProcess

For now I updated the nvdacompat document, which lists the status of add-ons regarding NVDA compatibility and the availability on the website, and mark this as compatible (coming soon). Coming soon reflects that the maintainer has send a pull request against this repo asking the inclussion on the website.

Regards

**Comment by lukaszgo1** [comment]
Hello @nvdaes  and thanks for your approval. I'm not really sure - can non translators join the translation system merely  to be able to update documentation? If so it seems to me these parts should be separated similar to what is being done for NVDA itself i.e English documentation should be kept in one repo and translators should be able to access only the files needed for translation. Since you have necessary permissions in the system I believe would it be possible for you to update the winWizard's readme when the PR gets merged - I'm quite short on time nowadays?

**Comment by nvdaes** [comment]
Hello: I"ll update the documentation on the website when this PR is merged. Cheers

**Pull request merged**
